### PR TITLE
[JVM] Fix contructors handling for JVM projects

### PR DESCRIPTION
**Pull request by arthurscchan**
This is an improvement for the JVM projects handling of oss-fuzz-gen after #236.

### CODE REVIEW

File: llm_toolkit/prompt_builder.py [hunk 1]
```diff
@@ -380,10 +380,14 @@ def __init__(self,
...
-    self.problem_template_file = self._find_template(template_dir,
-                                                     'jvm_problem.txt')
     self.requirement_template_file = self._find_template(
         template_dir, 'jvm_requirement.txt')
+    self.problem_template_file = self._find_template(template_dir,
+                                                     'jvm_problem.txt')
+    self.constructor_template_file = self._find_template(
+        template_dir, 'jvm_problem_constructor.txt')
+    self.method_template_file = self._find_template(template_dir,
+                                                    'jvm_problem_method.txt')
```
Comment (DavidKorczynski) [inline llm_toolkit/prompt_builder.py:390]: I can't find `jvm_problem_constructor.txt` and `jvm_problem_method.txt`, can you clarify?

**Review by DavidKorczynski — CHANGES_REQUESTED** [review]

File: llm_toolkit/prompt_builder.py (hunk 1 above)
Comment (arthurscchan) [reply llm_toolkit/prompt_builder.py:390]: Sorry those are missed in the commit....

**Review by DavidKorczynski — APPROVED** [review]

**Comment by DavidKorczynski** [comment]
Trial run conducted here: https://github.com/google/oss-fuzz-gen/pull/306
Results: https://llm-exp.oss-fuzz.com/Result-reports/ofg-pr/2024-06-07-306-test-jvm3-jvm/index.html

**Pull request merged**